tests/acceptance/features/bootstrap/DataExporterContext.php

```diff
@@ -256,14 +279,33 @@ private function assertFilePhysicallyExistInLastExport($filename, $message = '')
 	/**
 	 * @param string $filename
 	 * @param string $content
+	 * @param bool $trash
 	 *
 	 * @return void
 	 */
-	private function assertFilePhysicallyExistInLastExportWithContent($filename, $content) {
-		$this->featureContext->theFileWithContentShouldExistInTheServerRoot(
-			self::path("$this->lastExportPath/files/$filename"),
-			$content
-		);
+	private function assertFilePhysicallyExistInLastExportWithContent($filename, $content, $trash = false) {
+		if ($trash === true) {
+			$this->featureContext->listTrashbinFileInServerRoot(self::path("$this->lastExportPath/files_trashbin"));
+			$matches = [];
+			foreach (HttpRequestHelper::getResponseXml($this->featureContext->getResponse())->data->element as $item) {
+				$file = (string)$item;
+				if (strpos($file, $filename) === 0) {
+					$matches[] = $file;
+				}
+			}
+			foreach ($matches as $match) {
+				$this->featureContext->theFileWithContentShouldExistInTheServerRoot(
+					self::path("$this->lastExportPath/files_trashbin/$match"),
+					$content
+				);
```

Comment:
This tries to assert that the file content should match with every match found in the trashbin folder. But in our case we can just stop if one of the files with matching name has the same content. 
This will fail if we have multiple files with same name and different contensin the trashbin